i18n/de/docusaurus-plugin-content-docs/current/admin/02.what_is_yunohost/index.mdx

```diff
@@ -41,14 +41,6 @@ Der Name **YunoHost** stammt aus dem Jargon "Y U NO Host".
 ![The meme in question, showing a guy with a face bent in frustration and questioning "Y U NO Host"](/img/dude_yunohost.jpg)
 </center>
 
-## Was YunoHost nicht ist?
```

Comment:
(Will copypasta the new english version on other translations once English version is validated)